OpenRA.Game/Traits/World/Shroud.cs

```diff
@@ -383,13 +453,25 @@ public bool IsVisible(Actor a)
 
 		public bool IsTargetable(Actor a)
 		{
+			IEnumerable<CPos> vis = GetVisOrigins(a);
+			bool concealed = vis.All(c => generatedDisruptionCount[c] > 0 || generatedShroudCount[c] > 0);
```

Comment:
But the checks for regular actors and frozen actors have to be the same. Otherwise there are frozen actors that are visible but not auto-targetable. So either I have to check for footprints here or add the occupied cells to frozen actor.